CHANGELOG

```diff
@@ -1,3 +1,18 @@
+snowplow-ecommerce 0.7.2 (2024-03-04)
```

Comment:
Is there a particular reason it will be shipped only then?

Reply:
Honestly, gave me the rest of the week to work on it, but I got it done quicker than expected 😅